exercises/practice/protein-translation/tests/Tests.elm

```diff
@@ -0,0 +1,178 @@
+module Tests exposing (tests)
+
+import Expect
+import ProteinTranslation
+import Test exposing (Test, describe, skip, test)
+
+
+tests : Test
+tests =
+    describe "ProteinTranslation"
+        [ -- Tracks may choose to expect an error when an empty RNA sequence
+          -- is given, or use this test to expect an empty result.
```

Comment:
We usually don't keep those comments in, they are preserved by the generator for the exercise builder's sake.